client/src/pages/Detail/Detail.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { Col, Row, Container } from "../../components/Grid";
```

Comment:
A convention is growing in popularity where you separate module import from locally created file imports.